tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -1486,6 +1486,22 @@ public void MockDirectory_EnumerateDirectories_ShouldThrowWhenPathIsNotMocked()
             // Assert
             Assert.Throws<DirectoryNotFoundException>(action);
         }
+        
+        
+        [TestCase("Folder", "Folder/SubFolder")]
+        [TestCase("/Folder", "/Folder/SubFolder")]
```

Comment:
These test cases will only run under Linux and not under Windows.

Reply:
Yeah, I removed that one, since I have no idea how i can handle the root directory case in a platform-agnostic way.